paddle/fluid/pybind/imperative.cc

```diff
@@ -919,12 +920,17 @@ void BindImperative(py::module *m_ptr) {
               print(x.grad)          # None
        )DOC")
       .def("_run_backward",
-           [](imperative::VarBase &self, const imperative::Tracer &tracer,
-              bool retain_graph) {
+           [](std::shared_ptr<imperative::VarBase> &self,
```

Comment:
remove this method, call core._run_backward directly in python

Reply:
removed this method,  but core._run_backward is private which can not be find in python. use core.dygraph_run_backward instead.